__tests__/contact.test.jsx

```diff
@@ -0,0 +1,12 @@
+import "@testing-library/jest-dom"
```

Comment:
Couple of thoughts:
1. Let's rename the file to `footer.test.jsx`
2. I'd suggest using `.tsx` extension as the codebase is mainly typescript 
3. I would encourage moving the file closer to the source = `/components/footer.test.jsx`, wdyt?

Reply:
I'm not sure about moving the tests to the components folder (right beside the component itself) since some people might want to delete those tests, so it would be easier to just delete the whole folder. 
Though I suggest keeping them in __tests__/components/footer.test.tsx so that ti keeps a similar structure of the app or components folder.

Here's some examples inside __tests__:
/components
  footer.test.tsx
  navbar.test.tsx
  loader-button.test.tsx
/lib
  utils-calculations.test.tsx
  utils-common-functions.test.tsx

Reply:
Good point, let's follow your suggestion. I like it, let's do it 🙂